geomstats/geometry/euclidean.py

```diff
@@ -92,12 +109,14 @@ def exp(self, tangent_vec, base_point):
         Parameters
         ----------
         tangent_vec : array-like, shape=[..., dim]
-
+            Tangent vector.
```

Comment:
what about adding "at the base point"?

Reply:
Done, in this file and all files of `geometry`.